include/Core/UI/Component/DashboardWidget.php

```diff
@@ -44,7 +43,7 @@ public function render() {
 			<?php _e( 'Total number of emails logged' , 'email-log' ); ?>: <strong><?php echo number_format( absint( $logs_count ), 0, ',', ',' ); ?></strong>
 		</p>
 
-		<?php Util\render_auto_delete_logs_next_run_schedule(); ?>
+		<?php do_action( 'el_auto_delete_logs_next_run_schedule' ); ?>
```

Comment:
We should name hooks based on where they are present on not based on how they will be used.

Rename this to `el_inside_dashboard_widget`.

Reply:
Fixed.

Reply:
Fixed.